select_statements.sql

```diff
@@ -38,44 +38,31 @@ WHERE v.stationName = "Uppsala station" AND vc.name = "Stadsbil" AND (b.endDatum
 -- Underhållspersonal
 -- Sök fram alla bilar i behov av kontroll
 SELECT vehicle.registrationNumber FROM green_rental.vehicle
-LEFT JOIN booking_details ON vehicle.registrationNumber = booking_details.registrationNumber
+INNER JOIN booking_details ON vehicle.registrationNumber = booking_details.registrationNumber
 LEFT JOIN booking ON booking_details.bookingNumber = booking.bookingNumber
 LEFT JOIN control ON vehicle.registrationNumber=control.registrationNumber
-WHERE control.controlLarge is FALSE and control.datum < "2024-04-12" and booking.endDate = "2024-04-12";
+WHERE control.datum < "2024-04-12" and booking.endDate = "2024-04-12";
 
 -- Sök fram alla bilar i behov av kontroll inom 3/6/12 månader
-SELECT vehicle.registrationNumber
+SELECT vehicle.registrationNumber as Registreringsnummer
 FROM green_rental.vehicle
 LEFT JOIN control ON control.registrationNumber=vehicle.registrationNumber
-WHERE control.controlLarge is TRUE and control.datum>=DATE_SUB(CURDATE(), INTERVAL 3 MONTH);
+WHERE control.controlLarge is TRUE or control.controlDate is NULL and CURDATE()>=DATE_ADD(control.controlDate, INTERVAL 3 MONTH);
```

Comment:
I think what is even better is to use control.controlDate <= DATE_ADD(CURDATE(), INTERVAL 3 MONTH)

Reply:
Nice, fixed!